test/test_helper.rb

```diff
@@ -17,7 +17,7 @@
 require 'hammer_cli'
 require 'hammer_cli_foreman/testing/api_expectations'
 
-FOREMAN_VERSION = Gem::Version.new(ENV['TEST_API_VERSION'] || '1.20')
+FOREMAN_VERSION = Gem::Version.new(ENV['TEST_API_VERSION'] || '1.22')
```

Comment:
I have put this line to suggest it needs an upgrade, but I believe we should wait till the release of foreman to actually do it, right? I am doing it locally as of now.